model/userModel.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require("mongoose");
+const schema = mongoose.Schema;
+const userKey = new schema(
+  {
+    firstName: {
+      type: String,
+    },
+    lastName: {
+      type: String,
+    },
+    email: {
+      type: String,
+    },
+    password: {
+      type: String,
+    },
+  },
+  {
+    timestamps: true,
+  }
+);
+module.exports = mongoose.model("user", userKey, "user");
```

Comment:
add all logic in models

Reply:
userModal = mongoose.model("user", userKey, "user");

userModal['createUser'] = (body)=>{

}

module.exports = useMOdel;